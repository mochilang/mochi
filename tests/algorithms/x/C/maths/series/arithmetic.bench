{
  "duration_us": 444,
  "memory_bytes": 1671168,
  "name": "main"
}