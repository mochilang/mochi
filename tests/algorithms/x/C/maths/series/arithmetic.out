1
0
4
7.5